package/src/scripts/handles/handleInput.ts

```diff
@@ -4,19 +4,17 @@ import handleClick from '@scripts/handles/handleClick';
 import update from '@scripts/update';
 
 const setPositionCalendar = (input: HTMLInputElement, calendar: HTMLElement, position: 'left' | 'center' | 'right') => {
+	const inputRect = input.getBoundingClientRect();
+	const calendarRect = calendar.getBoundingClientRect();
+
 	const getPosition = {
-		left: 0,
-		center: input.offsetWidth / 2 - calendar.offsetWidth / 2,
-		right: input.offsetWidth - calendar.offsetWidth,
+		left: inputRect.left,
+		center: inputRect.left + inputRect.width / 2 - calendarRect.width / 2,
+		right: inputRect.right - calendarRect.width,
 	};
 
-	let top = input.offsetHeight;
-	let left = getPosition[position];
-
-	for (let el: HTMLElement = input; el; el = el.offsetParent as HTMLElement) {
-		top += el.offsetTop || 0;
-		left += el.offsetLeft || 0;
-	}
+	const top = inputRect.top + inputRect.height;
```

Comment:
it should also consider `scrollY`

```js
const top = window.scrollY + inputRect.bottom;
```

Reply:
Thanks for the information, fixed in v2.9.5